test/helpers/test.go

```diff
@@ -171,6 +177,55 @@ func (t *TestOptions) GetGitProvider() (gits.GitProvider, error) {
 	return gitProvider, nil
 }
 
+// GetApproverGitProvider returns a git provider that uses credentials for the approver user defined in environment variables
+// We don't use standard user auth here because the user/token isn't defined during boot, so we'll create the credentials
+// for the user on the fly.
+func (t *TestOptions) GetApproverGitProvider() (gits.GitProvider, error) {
```

Comment:
is there not a lot of cut&paste between `GetGitProvider` and `GetApproverGitProvider`. They only differ in which UserAuth to retrieve, right? Could one not at least extract a `GetAuthServer` of sorts?

Reply:
Yup, we can definitely simplify. I’ll do that momentarily.